src/foam/u2/ExportModal.js

```diff
@@ -103,12 +103,12 @@ foam.CLASS({
       var exportDriver = await this.exportDriverRegistryDAO.find(this.dataType);
       exportDriver = foam.lookup(exportDriver.driverName).create();
 
-      this.downloadCSV(this.exportData, exportDriver);
+      this.downloadCSVv(this.exportData, exportDriver);
     }
   ],
 
   listeners: [
-    function downloadCSV(data, exportDriver) {
+    function downloadCSVv(data, exportDriver) {
```

Comment:
Give this a different name. downloadCSV_ perhaps. Or move the body of this method into the downloadCSV function since it's the only caller of it.